src/main/resources/application.yml

```diff
@@ -0,0 +1,2 @@
+server:
```

Comment:
Cloud foundry is spring aware and should override this port to be 8080 when deployed.